tools/start_gcloud_and_exit.sh

```diff
@@ -0,0 +1,10 @@
+# script must be executed from root git directory
+
+# start_gcoud.sh is blocking. This script wraps it to exit as soon
+# as initialization is done (replace unreliable and slow wait call)
+rm start_gcloud.log
+touch start_gcloud.log
+(tools/start_gcloud.sh 2>&1 | tee start_gcloud.log) &
+{ tail -n +1 -f start_gcloud.log &} | sed -n '/Dev App Server is now running/q'
```

Comment:
Bug prone if the log line is changed, but I don't see a better solution. The aim of this comment is to raise the point in case one of us has a better solution.